base/precise-code-intel/api-server.Deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Precise code intelligence services for Sourcegraph
+  labels:
+    deploy: sourcegraph
+  name: pci-api-server
```

Comment:
I really dislike acronyms like `pci`. Such acronyms do not help anyone except those who already know what the thing is (same problem as LSIF).

I would be far happier with these being named `precise-code-intel-api-server`, `precise-code-intel-bundle-manager`, etc.

Reply:
Should pci be replaced everywhere in https://github.com/sourcegraph/deploy-sourcegraph/pull/574/files as well?

Reply:
Updated.